src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -83,19 +83,25 @@ public static Email parseEmail(String email) throws ParseException {
      * Parses a {@code String remark} into an {@code Remark}.
      * Leading and trailing whitespaces will be trimmed.
      */
-    public static Remark parseRemark(String remark) {
+    public static Remark parseRemark(String remark) throws ParseException {
         requireNonNull(remark);
         String trimmedRemark = remark.trim();
+        if (trimmedRemark.isEmpty()) {
+            throw new ParseException(RemarkCommandParser.MESSAGE_CONSTRAINTS);
+        }
         return new Remark(trimmedRemark);
     }
 
     /**
      * Parses a {@code String pendingQuestion} into an {@code pendingQuestion}.
      * Leading and trailing whitespaces will be trimmed.
      */
-    public static PendingQuestion parsePendingQuestion(String pendingQuestion) {
+    public static PendingQuestion parsePendingQuestion(String pendingQuestion) throws ParseException {
         requireNonNull(pendingQuestion);
         String trimmedPendingQuestion = pendingQuestion.trim();
+        if (trimmedPendingQuestion.isEmpty()) {
```

Comment:
LGTM! useful changes, as only allowing removing of remark and pq to be done using remove is probably less confusing for users